Expensify/App
# Add transition between Chats and Search views

### Details
cc @shawnborton wondering if you are into a transition like this? 
Was cleaning some things up and thought it could use a fade.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/152779

### Tests
1. Open app
2. Touch Magnifying Glass icon
3. Verify that the sidebar view fades to the Search view
4. Touch X 
5. Verify the sidebar view fades back to the Chats view

### Tested On


### Screenshots

#### Web
https://user-images.githubusercontent.com/32969087/106338186-42e59e80-6237-11eb-9099-a2504030c141.mp4

#### Mobile Web
https://user-images.githubusercontent.com/32969087/106342485-218baf00-6245-11eb-8059-581d99b99175.mp4

#### Desktop
https://user-images.githubusercontent.com/32969087/106342562-834c1900-6245-11eb-828e-a27e2f3acf0d.mp4

#### iOS
https://user-images.githubusercontent.com/32969087/106342359-9ad6d200-6244-11eb-8785-f3aff9f0c9bc.mp4

#### Android
https://user-images.githubusercontent.com/32969087/106342521-4e3fc680-6245-11eb-9f3b-c341438ba6dc.mp4
